platforms/sound/directsound/CustomSoundSystem.hpp

```diff
@@ -49,4 +49,6 @@ class SoundSystemDS : public SoundSystem
 	IDirectSound* m_directsound;
 	LPDIRECTSOUND3DLISTENER m_listener;
 	std::vector<BufferInfo> m_buffers;
-};
\ No newline at end of file
+};
+
+#define SOUND_SYSTEM SoundSystemDS
```

Comment:
> You can't really use DirectSound on non-Windows; nor can you use OpenSL ES on desktop Linux and Windows.

The issue is OpenAL. It can be used on both SDL and Win32, so you end up with:
```c++
#ifdef USE_OPENAL
#include "SoundSystemAL.h"
#define SOUND_SYSTEM SoundSystemAL
#else
#include "SoundSystemDS.h"
#define SOUND_SYSTEM SoundSystemDS
#endif
```
In both `platforms/sdl/main.cpp` and `platforms/windows/main.cpp`.

I'll switch to `typedef`s though, that is nicer.

Reply:
Yep, that's fine. I don't mind that at all.